version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "4.1.1"
+version in ThisBuild := "4.2.1"
```

Comment:
4.2.0 would be the version 😄

Reply:
That's right. I'll wait on this last small update and merge this in right after